taric_parsers/forms.py

```diff
@@ -0,0 +1,149 @@
+from typing import Sequence
+
+from crispy_forms_gds.helper import FormHelper
+from crispy_forms_gds.layout import Layout
+from crispy_forms_gds.layout import Size
+from crispy_forms_gds.layout import Submit
+from django import forms
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.files.uploadedfile import InMemoryUploadedFile
+from django.db import transaction
+
+from importer.models import BatchImportError
+from importer.models import ImportBatch
+from importer.namespaces import TARIC_RECORD_GROUPS
+from taric_parsers.chunker import chunk_taric
+from taric_parsers.importer import run_batch
+from workbaskets.models import WorkBasket
+from workbaskets.validators import WorkflowStatus
+
+
+class NewImportFormMixin:
```

Comment:
The "New" prefix won't age well, but I don't think any prefix is required anyway since it's namespaced by the taric_parsers app - it won't clash with a class of the same name in the importer module.
```suggestion
class ImportFormMixin:
```

Reply:
yep, thats done